airbyte-workers/build.gradle

```diff
@@ -15,7 +15,7 @@ configurations.all {
     resolutionStrategy {
         // Ensure that the versions defined in deps.toml are used
         // instead of versions from transitive dependencies
-        force libs.flyway.core, libs.jooq, libs.s3, libs.aws.java.sdk.s3
+        force libs.flyway.core, libs.jooq, libs.s3, libs.aws.java.sdk.s3, libs.jsonsmart
```

Comment:
Any particular reason to add `jsonsmart` lib?

Reply:
yes, the provided version has know vulnerabilities, so we bumped it and force it here. 

https://nvd.nist.gov/vuln/detail/CVE-2023-1370